tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -17,8 +17,16 @@ Documentation  Test 5-25 - OPS-User-Grant
 Resource  ../../resources/Util.robot
 Suite Setup  Wait Until Keyword Succeeds  10x  10m  Ops User Create
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+Test Teardown  Run Keyword And Ignore Error  Gather VC Logs
 
 *** Keywords ***
+
+Gather VC Logs
+    Log To Console  Collecting VC logs ..
+    Run Keyword And Ignore Error  Run  govc logs.download
+    Run Keyword And Ignore Error  Run  mv vcsupport*.tgz vcsupport.tgz.log
```

Comment:
presume this is so that the bundle gets gathered by the test harness? We already gather the container-logs bundle from the run (which seem to be .zip) - is it as simple as updating a pattern somewhere to grab .tgz as well? @mhagen-vmware

Reply:
Yep, the place for this would be https://github.com/vmware/vic/blob/master/tests/resources/VCH-Util.robot#L367
With an additional: `Run Keyword If  '%{HOST_TYPE}' == 'VC'  gather what logs you want`, however as you can see in that keyword already it pulls *specific* logs.  Otherwise, this govc logs.download would just add 5-15 minutes to every single test case which we probably don't want - esp as we are moving CI to VC which would hit that case every time as well. As we discussed in slack the current bundle inclusion pattern is .zip .log .debug... zip is very specific because that is the only archive extension that github allows attaching so if you are pulling the whole govc logs.download you really should just dump that archive into a zip archive I guess.

Reply:
If you just wanted to gather all the logs, we have that already here - although could be better named as it will work on either esx or vc:
https://github.com/vmware/vic/blob/master/tests/resources/Vsphere-Util.robot#L193